src/test/java/tests/RegistrationTest.java

```diff
@@ -52,7 +53,6 @@ public void registerNewUserTest() throws InterruptedException {
         signUpPage.inputRepeatedPassword(regData.getPassword());
         logger.info("Password has been repeated.");
         signUpPage.createAccountAccept();
-        Thread.sleep(10000);
-        Assert.assertEquals(logInPage.getSignInText(), "Sign In");
+        Assert.assertEquals(logInPage.getSignInText(), "to continue E-Restaurant");
```

Comment:
It better to write extra info about passing test after Assert. It can be more informative in the future.